src/components/Card/CardNumber/index.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import Styled from './index.style'
+
+interface CardNumberProps {
+  number1: string
+  number2: string
+  number3: string
+  number4: string
+}
+
+const CardNumber = ({
+  number1,
+  number2,
+  number3,
+  number4,
+}: CardNumberProps) => {
+  return (
+    <Styled.CardNumberContainer>
+      <Styled.CardTextContainer>{number1.slice(0, 4)}</Styled.CardTextContainer>
+      <Styled.CardTextContainer>{number2.slice(0, 4)}</Styled.CardTextContainer>
+      <Styled.CardTextContainer>
+        {(number3 ?? '').replace(/[0-9]/g, '*').slice(0, 4)}
+      </Styled.CardTextContainer>
+      <Styled.CardTextContainer>
+        {(number4 ?? '').replace(/[0-9]/g, '*').slice(0, 4)}
```

Comment:
```suggestion
      <Styled.CardTextContainer>{number1.}</Styled.CardTextContainer>
      <Styled.CardTextContainer>{number2}</Styled.CardTextContainer>
      <Styled.CardTextContainer>
        {(number3 ?? '').replace(/[0-9]/g, '*')}
      </Styled.CardTextContainer>
      <Styled.CardTextContainer>
        {(number4 ?? '').replace(/[0-9]/g, '*')}
```
input에서 충분히 숫자 길이를 제한하고 있으니 slice를 뺴도 괜찮을것 같습니다.